style.css

```diff
@@ -24,6 +24,152 @@ html {
   color: #1a2236;
 }
 
+/* -----------------------------POPUP CSS SECTION STARTS------------------------------------ */
+
+.popup__container {
+  width: 100%;
+  padding: 16px;
+  gap: 25px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: #fff;
+  border: #1a2236 28px solid;
```

Comment:
[OPTIONAL]
- [ ] For a better user experience, You may consider giving your modal a height that fits in the screen, no need to scroll down to see the whole content of it. or it would be great to place the scroll on the popup itself and not the main scroll bar of the body.